css/index.css

```
@@ -0,0 +1,239 @@
+
+.header {
+  display: flex;
+  background-color: #1f2937;
+  overflow: hidden;
+  height: 80px;
+}
+
+.center {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  align-content: space-between;
+}
+
+.logo {
+  height: 60px;
+  width: 252px;
+}
+
+.nav {
+  flex-grow: 1;
+  padding-left: 30px;
+}
+
+.nav__menu {
+  display: flex;
+}
+
+.nav__el {
+  padding: 0px 8px;
+}
+
+.nav__btn {
+  background: #1f2937;
+  border-radius: 6px;
+  border: none;
+  padding: 8px 12px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 14px;
+  color: #d1d5db;
+}
+
+.nav__btn:hover {
+  background: #42537a;
+  border-radius: 6px;
+  border: none;
+  padding: 8px 12px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 14px;
+  color: #d1d5db;
+}
+
+.active {
+  background: #111827;
+  border-radius: 6px;
+  border: none;
+  padding: 8px 12px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 14px;
+  color: white;
+}
+
+.profile {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  align-content: space-between;
+}
+
+.profile__name {
+  color: white;
+  padding-right: 5px;
+  font-size: 15px;
+}
+
+.profile__avatar {
+  background-color: #d1d5db;
+  border-radius: 50%;
+  height: 32px;
+  width: 32px;
+}
+
+.container__title {
+  font-size: 40px;
+  font-style: normal;
+  text-align: center;
+  padding: 65px 0px 65px 0px;
+}
+
+.article__info {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.article__subtitle {
+  font-weight: 700;
+  font-size: 32px;
+  padding-bottom: 50px;
+}
+
+.article__subtitleCenter {
```

```suggestion
.article__subtitle-center {
```
поменять все классы с таким наименованием, camelCase  не используем